File: Java_etc/jdbc/src/main/java/jdbc/model/BluRay.java
package jdbc.model;

public class BluRay extends Article {
	
	private Boolean troisD;

	public BluRay() {
	}

	public BluRay(Integer numeroArticle, Integer nbDisques, Boolean troisD) {
		super(numeroArticle, nbDisques);
		this.troisD = troisD;
	}

	public BluRay(Integer nbDisques, Boolean troisD) {
		super(nbDisques);
		this.troisD = troisD;
	}

	public Boolean getTroisD() {
		return troisD;
	}

	public void setTroisD(Boolean troisD) {
		this.troisD = troisD;
	}

	
}

File: angularCorrectionThibault/src/app/model/bluray.ts
import {Article} from './article';

export class Bluray extends Article {

  constructor(private _troisD: Boolean) {
    super();

  }


  get troisD(): Boolean {
    return this._troisD;
  }

  set troisD(value: Boolean) {
    this._troisD = value;
  }
}
